docs/index.html

```diff
@@ -25,7 +21,7 @@
 
     */
 
-    var content = { 
+    const content = { 
         defaultImage: "http://i.giphy.com/l0HlRnAWXxn0MhKLK.gif",
```

Comment:
```            if (elem.image) { 
￼                 var image = elem.image;
￼             } else {
￼                 var image = content.defaultImage;
￼             }```
There was a default image before, but you don't use it anymore. That's ok, the `defaultImage` can to be removed now.